map.html

```diff
@@ -62,48 +60,42 @@
 <div id="map"></div>
 <script>
     function initMap() {
-        map = new google.maps.Map(document.getElementById('map'), {
+        // initialize Leaflet
+        var mainLayer = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
```

Comment:
this source is all native languages, we might want one that is only english or english + native?

Reply:
That was apparently a decision by OSM to favour native languages on their primary map. There are freely available French- and German-language versions though, which I'll include as layers. There's a cycling-focused map that's mostly in English, but requires registration.